TestArrayGroup/src/TestDynamicData.F90

```diff
@@ -6,24 +6,21 @@ subroutine TestArrayGroup_DynamicDataF(CCTK_ARGUMENTS)
   DECLARE_CCTK_PARAMETERS
   DECLARE_CCTK_ARGUMENTS
 
+  integer, dimension(3) :: dim3
+
+  ! RANK() is not supported by all compilers, so we fail to complile here instead
```

Comment:
No need to add a comment on `RANK` if the code doesn't actually use `RANK`.

Reply:
right.